source/nodejs/adaptivecards-designer/src/card-designer-surface.ts

```diff
@@ -575,6 +584,33 @@ export class CardDesignerSurface {
             return !e.cancelBubble;
         }
 
+        this._designerSurface.onkeydown = (e: KeyboardEvent) => {
+            if (this._selectedPeer) {
+                if (e.ctrlKey && e.key == "b") {
+                    let buttons = this._peerCommandsHostElement.children;
+                    for (let i = 0; i < buttons.length; i++) {
+                        let button = buttons.item(i);
+                        if (button instanceof HTMLButtonElement) {
+                            if (button.innerText == "Bind...") {
+                                button.click();
+                            }
+                        }
+                    }
+                }
+                if (e.ctrlKey && e.key == "x") {
+                    let buttons = this._peerCommandsHostElement.children;
+                    for (let i = 0; i < buttons.length; i++) {
+                        let button = buttons.item(i);
+                        if (button instanceof HTMLButtonElement) {
+                            if (button.title == "Add an action") {
+                                button.click();
+                            }
+                        }
+                    }
+                }
+            }
```

Comment:
Do you really intend to invoke _every_ button that matches here?

If **"No"**, 

```suggestion
if (this._selectedPeer && e.ctrlKey) {
    const keyToText = {
        "b": "Bind...",
        "x": "Add an action"
    };

    if (keyToText[e.key]) {
        Array.from(this._peerCommandsHostElement.children).find(e => e instanceof HTMLButtonElement && e.innerText == keyToText[e.key]);
    }
}
```

(if **"Yes"**, check the other comment)